### ISSUE TITLE
Is this working for latest Svelte?

### ISSUE THREAD

**Issue by hmaesta** [henriquecaraujo/svelte-copy-to-clipboard#13]
Hello, Henrique.
Thanks for your contribution to Svelte community.

My question is... Is this package working for latest Svelte version?

I can't make it work even using the example presented in README file.

- REPL for README example: https://svelte.dev/repl/c59fdd4ca55e4c4fb942f31967dc0100?version=3.37.0

Example file doesn't work in Svelte REPL too. 

- REPL for `example/App.svelte`: https://svelte.dev/repl/475b38bd300c4ac0a1693b229b2cd20d?version=3.37.0

**Comment by Vanderscycle** [comment]
@hmaesta I have it working on the latest svelte kit version. You can find how I have it working [here](https://github.com/Vanderscycle/Professional-website/blob/4e487a69535ebd2b291250241c7035763d379ac5/src/components/ui/QrCode.svelte#L52)

**Comment by 0gust1** [comment]
yes, it still works perfectly with sveltekit@2.0.4, svelte@4.2.8

ping @henriquecaraujo
